core/templates/dev/head/services/CsrfTokenService.ts

```diff
@@ -19,37 +19,45 @@
 // This needs to be imported first instead of using the global definition
 // because Angular doesn't support global definitions and every library used
 // needs to be imported explicitly.
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
 import $ from 'jquery';
```

Comment:
Can we try to import only the `ajax` method from jquery?

Reply:
I did a google search for that..didn't find anything relevant